src/main/java/Wolox/training/models/Book.java

```diff
@@ -0,0 +1,120 @@
+package Wolox.training.models;
+
+import javax.persistence.*;
+
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
```

Comment:
When working with ORMs such as Hibernate, it is recommended to use the wrapper class for the type of the primary keys
```suggestion
    private Long id;
```